src/client/Microsoft.Identity.Client/Platforms/netcore/NetCorePoPCryptoProvider.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Cryptography;
+using Microsoft.Identity.Client.AuthScheme.PoP;
+using Microsoft.Identity.Client.PlatformsCommon;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client.Platforms.netcore
+{
+    /// <summary>
+    /// The default implementation will store a key in memory for a period of 8 hours and recreate it to ensure the key is kept fresh.
+    /// Subsequent calls to this provider will reuse this key.
+    /// </summary>
+    internal class NetCorePoPCryptoProvider : IPoPCryptoProvider, IDisposable
```

Comment:
You moved a bunch of code outside, but I am wondering if there is any difference between NetCorePoPProvider and NetDesktopPopProvider?

Reply:
I looked at this in more detail, I think you can merge these 2 classes toghter and remove the POPHelper